src/torrent.cpp

```diff
@@ -8244,16 +8244,16 @@ namespace libtorrent
 			m_seeding_time += lost_seconds;
 		}
 		m_became_seed = clamped_subtract_u16(m_became_seed, seconds);
-
-		if (m_finished_time < seconds && is_finished())
+		if (m_finished_time < seconds && is_finished() && !is_paused())
```

Comment:
I guess I need to create another test where we test time shift with an active torrent. I could use 1 node with unlimited online time like you did on RC_1_1 because I don't need to test upload time it would be enogh to test finish time. But I guess it would still need 30s execution time to get 70000 ticks executed. We will see.